### ISSUE TITLE
Inconsistent use of images`

### ISSUE THREAD

**Issue by JimDaly** [MicrosoftDocs/powerapps-docs#358]
Why does this topic include links to a list of figures rather than use inline images?
The Azure application registration has links to images at the bottom.
Makes the procedure hard to follow

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f0f87a0a-9dd9-5188-7347-d910423902ca
* Version Independent ID: 9f532b04-f9c5-cbdd-a2bd-666b55ee942e
* Content: [Use Single-Tenant server-to-server authentication (Common Data Service) - PowerApps](https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/use-single-tenant-server-server-authentication#bkmk_app-registration-started)
* Content Source: [powerapps-docs/developer/common-data-service/use-single-tenant-server-server-authentication.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/developer/common-data-service/use-single-tenant-server-server-authentication.md)
* Service: **powerapps**
* GitHub Login: @paulliew
* Microsoft Alias: **jdaly**

**Comment by paulliew** [comment]
… because the contents/sources belong to Azure AD, and if we include inline images, the contents can get outdated pretty fast.

**Comment by phecke** [comment]
@JimDaly Images have been moved inline for the next time we publish to the website.
